scripts/config.pl

```diff
@@ -63,6 +63,7 @@
                               excluding some reserved symbols, until the
                               'Module configuration options' section
     realfull                - Uncomments all #define's with no exclusions
+    baremetal               - Set configuration suitable for baremetal build.
```

Comment:
Please clarify that it sets a *full* configuration suitable for baremetal builds. (Also, while I prefer `Set` to `Sets` I think the latter would be more consistent with the rest of the help message here.)